ui/hooks/useMultichainAccountsIntroModal.ts

```diff
@@ -33,17 +32,24 @@ export function useMultichainAccountsIntroModal(
   );
 
   const lastUpdatedAt = useAppSelector((state) => state.metamask.lastUpdatedAt);
-  const lastUpdatedFromVersion = useAppSelector(getLastUpdatedFromVersion);
+  const lastUpdatedFromVersion = useAppSelector(
+    (state) => state.metamask.previousAppVersion,
```

Comment:
For other reviewrs: we use this version instead, seems it's guaranteed to be semver-compatible, while the `getLastUpdatedFromVersion` uses a different format (which is one of the reason this hook fails...)